extensions/db-diagram/src/models/Generics/INetworkGraphModel.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import INodeModel from './INodeModel';
+
+
+export default interface INetworkGraphModel {
+	name: string;
+	summary: string;
+	nodes: INodeModel[];
+	metadata: JSON;
```

Comment:
Is a JSON blob the best type here? Any reason not to use a dictionary of key/value pairs instead or do you think you'll actually have nested objects?